src/inc/til/string.h

```diff
@@ -48,4 +48,26 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
     {
         return starts_with<>(str, prefix);
     };
+
+    template<typename T, typename Traits>
+    constexpr bool ends_with(const std::basic_string_view<T, Traits> str, const std::basic_string_view<T, Traits> prefix) noexcept
+    {
+#ifdef __cpp_lib_ends_ends_with
+#error This code can be replaced in C++20, which natively supports .ends_with().

```

Comment:
Is it more meaningful to smack developers who try to switch to C++20, or simply call `ends_with` ourselves? :wink:

Reply:
SMACK 'EM! 👊

Jokes aside: Maybe a `#warning`? The idea behind the `#error` is that if anyone ever upgrades to C++20, they should be able to remove the `#error` line, if they thing that's appriopriate.
What's your opinion about this?